test/test_sparse.py

```diff
@@ -2386,7 +2386,7 @@ def test_sparse_to_numpy(self):
         t = torch.sparse_coo_tensor(torch.tensor(([0, 0], [2, 0])), torch.tensor([1, 4]))
         self.assertRaises(TypeError, lambda: t.numpy())
 
-    @cpu_only
+    @skipIfRocm
```

Comment:
Why is the skip on ROCm necessary?

Reply:
I skipped ROCM as in other sparse functions  are skipped also.  Just in case let me try .

Reply:
I tested with ROCm enabled and It can't run the second test.  
https://ci.pytorch.org/jenkins/job/pytorch-builds/job/pytorch-linux-bionic-rocm3.7-py3.6-test2/1903/console

As it is not related specifically to softmax, I will enable again this decorator.